### PR TITLE
Updated changes for touch UI

### PR DESCRIPTION
**Pull request by mohansgithub**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -13,8 +13,8 @@
 			<h1>DevOps Traning Mohanraj Vellingiri</h1>
 			<h1>Devops Training Nallaperumal TN	</h1>
 			<h1>Devops Training Nallaperumal TN	</h1>
-			<h1>DevOps Training Mohit Kambam</h1>
-			<h1>UI is developed by Ramesh	</h1>
+			<h1>Updated by Mohanraj</h1>
+			<h1>Change for TouchScreen UI	</h1>
 			<h1>DevOps Training Mohit Kambam</h1>
```
Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:18]: Look Good to me

Comment (mydevopstrainings1) [inline src/main/webapp/WEB-INF/views/home.jsp:18]: Update Vijay Name

**Review by mydevopstrainings1 — CHANGES_REQUESTED** [review]
Please Updated

File: src/main/webapp/WEB-INF/views/home.jsp (hunk 1 above)
Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:18]: OK

Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:18]: Updated as per requested

Comment (mydevopstrainings1) [reply src/main/webapp/WEB-INF/views/home.jsp:18]: ok

**Review by mydevopstrainings1 — APPROVED** [review]
I Approve the same

**Pull request merged**